PartEzApp/app/src/main/java/com/partez/EventDetailsActivity.java

```diff
@@ -97,7 +97,7 @@ private JSONArray getJsonEditTextOutput(LinearLayout layout, String name) throws
         for(int i = 0; i < layout.getChildCount(); i++)
         {
             View v = layout.getChildAt(i);
-            // get comma separated list of emails
+
```

Comment:
Could we take this change out so that we can skip the conflict with my PR? Please and thanks! Other than that, looks great.